resources/instance_image_hook.rb

```diff
@@ -8,15 +8,13 @@
 
 action :enable do
   file ::File.join(config_dir, 'hooks', new_resource.name) do
-    mode 0755
-    action :touch
```

Comment:
This action needs to stay as it's not the default action. Same for below.

Reply:
I was running into idempotency issues if the action was touch for multi-converge

```
       Running handlers:
       First chef run should have reached a converged state.
       Resources updated in a second chef-client run:
       - instance_image_hook[create]
       - instance_image_hook[grub]
       - instance_image_hook[foo]
       - instance_image_hook[disable]
       - instance_image_hook[source]
```

It looks like touch is updating the utime on the files
```
       * instance_image_hook[source] action create
         * file[/etc/ganeti/instance-image/hooks/source] action touch
           - update utime on file /etc/ganeti/instance-image/hooks/source
```

Reply:
Hmm, you're right. I'll double check this works as expected when I run a local test.